tools/make_patch.c

```diff
@@ -108,9 +108,21 @@ int create_symbol(struct symbol **tip, const int value, const char *name)
 const struct symbol *find_symbol(const struct symbol *symbols, const char *name)
 {
 	const struct symbol *symbol = symbols;
+	int namelen = strlen(name);
+	int symbolnamelen = 0;
+	const char *equalLastCharacters = NULL;
 	while (symbol) {
-		if (strcmp(symbol->name, name) == 0) {
-			return symbol;
+		symbolnamelen = strlen(symbol->name);
```

Comment:
Just define the variable here, as in `int symbolname_len`

Reply:
Resolved in https://github.com/pret/pokecrystal/pull/882/commits/4ecffb8e22d745b6264cc2201cd39b212e78e46b